api-documentation/issue-api.yaml

```diff
@@ -184,11 +184,11 @@ paths:
             'appliation/json':
               schema:
                 $ref: '#/components/schemas/verifyResponse'
-  /revocation-list:
+  /credentials/revocation-list:
```

Comment:
Isnt the credential part of the base url?

Reply:
No @srprasanna, credentials is not part of the baseURL. We added this endpoint in the credential services so had to add the credentials.